src/codegen/solana_accounts/account_management.rs

```diff
@@ -118,11 +131,54 @@ fn process_instruction(instr: &mut Instr, functions: &[Function], ast_no: usize)
 
         *address = None;
         *accounts = Some(metas_vector);
+    } else if let Instr::AccountAccess { loc, name, var_no } = instr {
```

Comment:
Why is this not just a `match` on `instr`?

Reply:
I though about that too, but as I only had two cases I considered a match unnecessary.